nibabel/streamlines/tests/test_tractogram.py

```diff
@@ -570,6 +723,28 @@ def test_tractogram_to_world(self):
         tractogram.affine_to_rasmm = None
         assert_raises(ValueError, tractogram.to_world)
 
+    def test_tractogram_extend(self):
+        # Load tractogram that contains some metadata.
+        t = DATA['tractogram'].copy()
+
+        for op, in_place in ((operator.add, False), (operator.iadd, True),
+                             (extender, True)):
+            first_arg = t.copy()
```

Comment:
No, because `.copy()` is doing a deepcopy (https://github.com/nipy/nibabel/blob/master/nibabel/streamlines/tractogram.py#L346).